src/routes/channel.js

```diff
@@ -51,29 +51,61 @@ router.post(
 
 /**
  * @route GET api/channels
- * @access Public
+ * @access Private
  * @desc Get all channels
  */
 router.get('/', async (req, res) => {
   try {
-    let allChannels = await Channel.find();
+    await Channel.find()
+      .populate('users')
+      .exec((err, channels) => {
+        channels.map(channel => {
+          return {
+            name: channel.name,
+            description: channel.description,
+            dateCreated: channel.date,
+            id: channel._id,
+            users: channel.users,
+            comments: channel.comments,
```

Comment:
This whole return is not changing the response at all

Reply:
I just worked off your old setup and it works, take a look at my branch `comments-added-to-respective-channels`